### PR TITLE
feat(EMI-2099): task acknowledgment

### PR DESCRIPTION
**Pull request by leamotta**
https://artsyproduct.atlassian.net/browse/EMI-2099

Opening https://github.com/artsy/gravity/pull/18248
Similar to #6078

### CODE REVIEW

File: src/schema/v2/schema.ts [hunk 1]
```diff
@@ -238,6 +238,7 @@ import { DiscoverArtworks } from "./discoverArtworks"
 import { CreateDiscoveryLikedArtworkMutation } from "./createDiscoveryArtworkReferenceMutation"
 import { Invoice } from "./Invoice/invoice"
 import { createInvoicePaymentMutation } from "./Invoice/createInvoicePaymentMutation"
+import { ackTaskMutation } from "./me/ack_task_mutation"
```
Comment (MrSltun) [inline src/schema/v2/schema.ts:241]: Nitpick: can we change the name of the file to follow a Camel Case convention? just to be similar to the other files 👀? 
```suggestion
import { ackTaskMutation } from "./me/ackTaskMutation"
```

**Review by MrSltun — APPROVED** [review]
Looks good! I just left a small comment on the file name 👀

File: src/schema/v2/schema.ts (hunk 1 above)
Comment (leamotta) [reply src/schema/v2/schema.ts:241]: there's plenty of others that use this convention, including the dismiss mutation. Is the recommended format CamelCase? Should I change the dismiss mutation as well?

Comment (MrSltun) [reply src/schema/v2/schema.ts:241]: Maybe that would be great to follow one convention 😅 but it's not part of this PR's scope, so feel free to ignore the other ones

**Pull request merged**